app/views/addons/showaddon.html

```diff
@@ -0,0 +1,27 @@
+<ol class="breadcrumb">
+  <li><a ng-href="#/tvshows">TV Shows</a></li>
+  <li><a ng-href="#/tvshows/{{ season.tvshowId }}">{{ season.tvshow }}</a></li>
+  <li class="active">Season {{ season.season }}</li>
+</ol>
```

Comment:
Same here, the header and breadcrumb needs to be updated to reflect the Addons section